src/components/Map/index.js

```diff
@@ -28,10 +38,32 @@ class Map extends React.Component {
     if (this.props.hideZoomControls) {
       this.map.zoomControl.remove();
     }
+
+    this.renderMarker(this.props.latlng.latitude, this.props.latlng.longitude);
+  }
+
+  componentDidUpdate(prevProps) {
+    const lat = this.props.latlng.latitude;
+    const lng = this.props.latlng.longitude;
+    if (lat !== prevProps.latlng.latitude || lng !== prevProps.latlng.longitude) {
```

Comment:
hard to read. Maybe `if(!(lat == prevProps.latlng.latitude && lng == prevProps.latlng.longitude))`